warp/yul/ToCairoVisitor.py

```diff
@@ -155,11 +181,9 @@ def visit_variable_declaration(self, node: ast.VariableDeclaration) -> str:
             return value_repr
         vars_repr = ", ".join(f"local {self.visit(x)}" for x in node.variables)
         if isinstance(node.value, ast.FunctionCall):
-            if (
-                value_repr == "Uint256(31597865,9284653)"
-                or value_repr == "Uint256(0,0)"
-            ):
-                return f"{vars_repr} = {value_repr}"
+            if "calldatasize" in node.value.function_name.name:
```

Comment:
Shouldn't it be automatically translated via the `BultinHandler` mechanism? Why do we need to hardcode this?